apps/nostr-debug/NostrDebug.Web/Pow/PowDelegate.cs

```diff
@@ -0,0 +1,182 @@
+using System;
+using System.Security.Cryptography;
+
+namespace NostrDebug.Web.Pow
```

Comment:
I'm not really sure, but why do we have a second implementation of the PoW instead of calling the implementation on the NostrEvent?